Assets/Scripts/Models/GameModel.cs

```diff
@@ -0,0 +1,55 @@
+using UnityEngine;
+using Player;
+using Weapon;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Game {
+  public class GameModel {
+
+    public static int MAX_PLAYER_CAPCITY = 100;
+
+    private ArrayList Players = new ArrayList();
+    private ArrayList Weapons = new ArrayList();
+    private double Duration;
+    private DateTime StartTime;
+
+
+    public GameModel(float duration) {
+      this.Duration = duration;
```

Comment:
I don't think `this` is needed because the class properties are PascalCase and method arguments are camelCase

Reply:
I am used to this notation because I think specifying `this.` helps others understand the code more clearly. But I can remove this as well.